.github/workflows/checks.yml

```diff
@@ -54,7 +54,9 @@ jobs:
         with:
           python-version: 3.7
       - name: Install prerequisites
-        run: python scripts/gha/install_prereqs_desktop.py
+        run: |
+          python scripts/gha/install_prereqs_desktop.py
+          python3 -m pip install unidiff
```

Comment:
Is there a need to make this binary `python3`. I thought all of our `python` executions in GHA were python3 since that's the target our scripts were meant for.

Reply:
Good call (though the call to format_code above also seems to use python3)